### DISCUSSION TITLE
how to disable auto login?

### DISCUSSION THREAD

**Discussion by Ieh47** [minios-linux/minios-live#16] [General]
I tried to disable auto login with the comand:
sudo nano /etc/lightdm/lightdm.conf
Didn't work. How can I disable it?

**Comment by FershoUno** [comment]
hello @Ieh47 ,  to disable autologin, you need to configure it from the minios configuration file
**/etc/minios/minios.conf**, change "true" to "false"

`AUTOLOGIN="true"`

Review all options in the documentation
https://minios.dev/docs/docs/configuration/configuration-file/

**Reply by Ieh47** [reply]
Thank you so much. You helped me a lot

I also changed the USER_NAME and USER_PASSWORD, but the system crashed, then I changed  to the original USER_NAME again, and the system runs normal now. 
So it's possible to change just the USER_PASSWORD.

The only problem is that  if I press CTRL + ALT + F1 or CTRL + ALT + F2 on the login screen, pop up a screen that shows my user login name and my password 🥲

Edit: To solve this problem showing my username and password on the tty screen,  I changed 
HIDE_CREDENTIALS="false" 
To 
HIDE_CREDENTIALS="true"

Everything was in the link you gave me, so I have to say thank you one more time

**Reply by FershoUno** [reply]
It is possible to change the name and password, but keep in mind that it should not have symbols that bash could misinterpret:

For example:

```
USER_NAME="fernando"
USER_PASSWORD="SecureMyKey"
ROOT_PASSWORD="CalveR00t"
```


I think that in the next versions these details are being improved :D

**Reply by Ieh47** [reply]
The user name I tried was "ieh" , so I don't know what went wrong "/

**Reply by FershoUno** [reply]
umm, try a slightly longer name (minimum 4 characters)

**Reply by crim50n** [reply]
> I also changed the USER_NAME and USER_PASSWORD, but the system crashed, then I changed to the original USER_NAME again, and the system runs normal now.
> So it's possible to change just the USER_PASSWORD.

Perhaps in a future version of MiniOS it will be possible to change the user name as well. In version 3.2 it is only possible to change the user name before the first system boot.
